crates/iota-sdk/src/lib.rs

```diff
@@ -644,3 +636,87 @@ impl DataReader for ReadApi {
         Ok(self.get_reference_gas_price().await?)
     }
 }
+
+pub trait GetAllPages<O, C, F, E>: Sized + Fn(Option<C>) -> F
```

Comment:
How annoying would it be to write a test for it?

Reply:
I guess it's tested through `test_get_owned_objects_owned_by_address_and_check_pagination`, maybe that's enough